js/models/SMTPIntegrationSettings.js

```diff
@@ -0,0 +1,43 @@
+import BaseModel from './BaseModel';
+import app from '../app';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      smtpNotifications: false,
+      smtpServerAddress: 'defaultservername',
+      smtpUserName: '',
+      smtpPassword: '',
+      smtpFromEmail: '',
+      smtpToEmail: ''
+    };
+  }
+
+  validate(attrs) {
+    const errObj = {};
+    const addError = (fieldName, error) => {
+      errObj[fieldName] = errObj[fieldName] || [];
+      errObj[fieldName].push(error);
+    };
+
+    if (attrs.smtpNotifications) {
```

Comment:
@rmisio okay. I think the spec was to only save the fields if notifications are on. Tho maybe that doesn't make so much sense as what you just said. Agree validating anyway makes sense.